source/index/migrations/1_initial_migration.js

```diff
@@ -1,5 +1,7 @@
-const Migrations = artifacts.require('Migrations')
+const Index = artifacts.require('Index')
 
-module.exports = deployer => {
-  deployer.deploy(Migrations)
-}
+module.exports = (deployer, network) => {
+  if (network == 'rinkeby') {
+    deployer.deploy(Index)
```

Comment:
This would never be called.

Reply:
I was told one day we might want to have Indexes independent from an Indexer?